play/src/front/Components/MapEditor/AreaEditor/AreaPropertiesEditor.svelte

```diff
@@ -769,7 +769,7 @@
     }
 
     .property-box {
-        margin-top: 5px;
+        margin-top: 35px;
```

Comment:
This and `mt[3rem]` do the same thing. Shall we get rid of the CSS and keep Tailwind only?